docs/gcc-Tips.rst

```diff
@@ -0,0 +1,44 @@
+Recommendations When Using gcc
+==============================
+
+It is recommended to use ``-O3 -mtune=native`` to achieve maximum speed during LightGBM training.
+
+Using Intel Ivy Bridge CPU on 1M x 1K Bosch dataset, the performance increases as follow:
+
++-------------------------------------+---------------------+
+| Compilation Flag                    | Performance Index   |
++=====================================+=====================+
+| ``-O2 -mtune=core2``                | 100.00%             |
++-------------------------------------+---------------------+
+| ``-O2 -mtune=native``               | 100.90%             |
++-------------------------------------+---------------------+
+| ``-O3 -mtune=native``               | 102.78%             |
++-------------------------------------+---------------------+
+| ``-O3 -ffast-math -mtune=native``   | 100.64%             |
++-------------------------------------+---------------------+
+
+You can find more details on the experimentation below:
+
+-  `Laurae++/Benchmarks <https://sites.google.com/view/lauraepp/benchmarks>`__
```

Comment:
@Laurae2 Is new url https://sites.google.com/view/lauraepp/new-benchmarks/old-benchmarks ?

Reply:
Yes this is the correct URL

Reply:
Thanks